utils/algorithms.cpp

```diff
@@ -2,6 +2,8 @@
 
 using jbrush::Comparator;
 
+namespace jbrush {
```

Comment:
Don't think it is necessary.

Reply:
Are you suuuuure? 'Cause I didn't put that there and I don't want to delete it just for whoever put it there to yell at me and tell me to put it back

Reply:
Well, if you look at the ``git blame``, you will see that you have added these lines on commit with the message ``Did David's easier fixes. Plan on implementing harder stuff like keyb…``. I have no idea why it was added.

Reply:
Well, I must admit, I do NOT remember that! Anyways, I'll get rid of it

Reply:
Wait, actually I remember why it was added.  Because all of the prototypes in the `.hpp` are in the `jbrush` namespace.  Pretty sure that was already there when I rebased the master, but I don't remember for sure.  Should I remove the namespace from the `.hpp` or put it back in the `.cpp`?